update/ansible/playbook/tasks/roles/grafana/tasks/main.yml

```diff
@@ -3,67 +3,34 @@
   file:
     path: "{{ item }}"
     state: directory
-    owner: grafana
-    group: grafana
-    mode: "0775"
+    owner: pmm
+    group: pmm
+    mode: 0755
   loop:
     - /srv/grafana
     - /srv/grafana/plugins
 
-- name: Set Grafana folder for plugins on /srv partition for all users
-  lineinfile:
-    path: /etc/bashrc
-    line: "export GF_PLUGIN_DIR=/srv/grafana/plugins"
+- name: Set ownership on /etc/grafana
+  file:
+    path: /etc/grafana
+    state: directory
+    owner: pmm
+    group: pmm
+    mode: 0744
```

Comment:
should we change mode for existing files?

Reply:
Yes, I was surprised to learn that grafana would not start otherwise. It was OK with the root user, but it's not when we run it as `pmm` user.